tools/iree-tracy-print.cc

```diff
@@ -0,0 +1,304 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+#include <atomic>
+#include <fstream>
+#include <iostream>
+#include <chrono>
+#include <map>
+#include <thread>
+
+#include "iree/base/internal/flags.h"
+#include "build_tools/third_party/tsl/xplane.pb.h"
+#include "third_party/tracy/public/common/TracyProtocol.hpp"
+#include "third_party/tracy/server/TracyFileRead.hpp"
+#include "third_party/tracy/server/TracyFileWrite.hpp"
+#include "third_party/tracy/server/TracyWorker.hpp"
+
+IREE_FLAG(string, input_tracy_file, "",
+          "Tracy file to read. Ignored if executable command is given.");
+IREE_FLAG(string, output_tracy_file, "",
+          "Tracy file to write as the output of the given executable command.");
+IREE_FLAG(string, output_xplane_file, "",
+          "Xplane file to write as the output of execution or conversion.");
+IREE_FLAG(int32_t, tracy_port, 18086, "TCP port number for tracy profiler.");
+
```

Comment:
With the name change, we would need an output option for printing since pinting is a kind of output.

Reply:
Done